core-bundle/contao/library/Contao/StringUtil.php

```diff
@@ -208,6 +208,11 @@ public static function substrHtml($strString, $intNumberOfChars)
 	 */
 	public static function decodeEntities($strString, $strQuoteStyle=ENT_QUOTES)
 	{
+		if (\is_array($strString))
+		{
+			return array_map(__METHOD__, $strString, $strQuoteStyle, $strCharset);
```

Comment:
`$strQuoteStyle` causes `Uncaught TypeError: array_map(): Argument #3 must be of type array, int given`

`$strCharset` causes `Undefined variable $strCharset`